### PR TITLE
add escape requests in azimuth

### PR DESCRIPTION
**Pull request by liam-fitzgerald**
Add getEscapeRequests and getEscapeRequestsCount into azimuth in preparation for bridge work.

### CODE REVIEW

**Comment by liam-fitzgerald** [comment]
cc: @jtobin

File: azimuth.js [hunk 1]
```diff
@@ -234,6 +234,22 @@ function getEscapeRequest(contracts, point) {
...
+ * @param {Number} point - Point number.
+ * @return {Promise<Array<Number>>} An array of points requesting escape
+ */
+const getEscapeRequests = internal.getEscapeRequests;
+
+/**
+ * Get the number of points that are requesting escape to a sponsor.
+ * @param {Object} contracts - An Urbit contracts object.
+ * @param {Number} point - Point number.
+ * @return {Promise<Array<Number>>} THe number of points requesting escape
```
Comment (Fang-) [inline azimuth.js]:
```suggestion
 * @return {Promise<Number>} THe number of points requesting escape
```

**Review by Fang- — APPROVED** [review]
Small documentation error, but looks good otherwise. (In the js context the count function is a bit superfluous since you can just `.length` the array, but not opposed to having it in for completeness.)

File: azimuth.js (hunk 1 above)
Comment (jtobin) [inline azimuth.js]:
```suggestion
 * @return {Promise<Array<Number>>} The number of points requesting escape
```

Comment (jtobin) [inline azimuth.js]:
```suggestion
 * @return {Promise<Array<Number>>} The number of points requesting escape
```

File: azimuth.js [hunk 2]
```diff
@@ -234,6 +234,22 @@ function getEscapeRequest(contracts, point) {
...
+ * @param {Number} point - Point number.
+ * @return {Promise<Array<Number>>} An array of points requesting escape
+ */
+const getEscapeRequests = internal.getEscapeRequests;
+
+/**
+ * Get the number of points that are requesting escape to a sponsor.
+ * @param {Object} contracts - An Urbit contracts object.
+ * @param {Number} point - Point number.
+ * @return {Promise<Number>} THe number of points requesting escape
```
Comment (jtobin) [inline azimuth.js]:
```suggestion
 * @return {Promise<Number>} The number of points requesting escape
```

**Pull request merged**